app/src/main/java/com/whereuat/whereu/MainActivity.java

```diff
@@ -111,16 +111,17 @@ public void onReceive(Context context, Intent intent) {
     }
 
     public void addContact(View view){
-
         // Check the SDK version and whether the permission is already granted or not.
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && checkSelfPermission(Manifest.permission.READ_CONTACTS) != PackageManager.PERMISSION_GRANTED) {
             requestPermissions(new String[]{Manifest.permission.READ_CONTACTS}, PERMISSIONS_REQUEST_READ_CONTACTS);
         } else {
-            Intent intent= new Intent(Intent.ACTION_PICK,  ContactsContract.Contacts.CONTENT_URI);
+            Intent intent= new Intent( Intent.ACTION_PICK, ContactsContract.Contacts.CONTENT_URI);
```

Comment:
Lol, no put a space between `intent` and `=` like this `intent = new` and get rid of this space `Intent( Intent`